alibi_detect/utils/_random.py

```diff
@@ -0,0 +1,80 @@
+from contextlib import contextmanager
+import random
+import numpy as np
+import os
+import sys
+from alibi_detect.utils.frameworks import has_pytorch, has_tensorflow
+
+# Pointer to the alibi_detect module. Used to set a global seed variable.
+_module_pointer = sys.modules[__name__]
+setattr(_module_pointer, '_seed', None)
```

Comment:
Not sure why this roundabout way instead of just `SEED = None` and then refer to `global SEED` in functions?

Reply:
Perhaps also a more clear vairable name and docstring, e.g. `ALIBI_SEED` or something like that.

Reply:
You can never truly hide anything in Python... I'm not so sure it's worth the cognitive overhead of interpreting these lines rather than just having a global variable with a clear name (can make it private too e.g. `_ALIBI_SEED`).